src/date.rs

```diff
@@ -13,13 +13,13 @@ use core::{fmt, hash};
 #[cfg(feature = "rkyv")]
 use rkyv::{Archive, Deserialize, Serialize};
 
+use crate::duration::Duration as OldDuration;
```

Comment:
Please add a user-friendly explanation for why `use crate::duration::Duration as OldDuration;` is done here. In other files it's as-is; `use crate::duration::Duration;`. It's confusing that the same type inexplicably changes names in some places.

```rust
// OldDuration is an unfortunate inherited name.
// It does not imply chrono::duration::Duration type should not be used.
// See Issue #1073
use crate::duration::Duration as OldDuration;
```

Obviously, this is a verbose comment to include everywhere there is an `as OldDuration`. Perhaps you can come up with something more concise. Either way, this seemingly random naming oddity should be addressed or it'll be very confusing.

Please do this for all `... as OldDuration;` statements.

Reply:
It should be addressed in a code comment. As-is, having `crate::duration::Duration` treated differently in different files is confusing.

Reply:
I added a short comment to the imports that this is to avoid confusion with `Duration` from the standard library in the docs.